project/.travis/before_script_test.sh.twig

```diff
@@ -1,3 +1,11 @@
 #!/usr/bin/env bash
 set -ev
 
+{% if repository_name is same as('SonataDoctrinePhpcrAdminBundle') %}
```

Comment:
do we have access to `repository_name` here @Soullivaneuh ?

Reply:
more then on `package_name` :-) i called the `dev-kit` in `dry-run` localy (i know that thing as i cloned it for cmf)

Reply:
Fine for me